tests/sentry/interfaces/test_stacktrace.py

```diff
@@ -9,11 +9,19 @@
 from exam import fixture
 
 from sentry.interfaces.base import InterfaceValidationError
-from sentry.interfaces.stacktrace import (Frame, Stacktrace, get_context, slim_frame_data)
+from sentry.interfaces.stacktrace import (Frame, Stacktrace, get_context, slim_frame_data, is_url)
 from sentry.models import Event
 from sentry.testutils import TestCase
 
 
+def test_is_url():
+    assert is_url('http://example.org/') is True
+    assert is_url('https://example.org/') is True
+    assert is_url('file:///tmp/filename') is True
+    assert is_url('applewebdata://00000000-0000-1000-8080-808080808080') is True
+    assert is_url('data:,') is False  # no authority
```

Comment:
I don't have a strong opinion about this last one but it's a lot simpler to parse if we're only concerned with URLs that contain an "authority" component (optional authentication, host, and optional port after a double slash.)